packages/lesswrong/lib/eaGivingSeason.ts

```diff
@@ -104,3 +104,112 @@ export const timelineSpec: TimelineSpec = {
     },
   ],
 };
+
+// These are charities/orgs that are *not* election candidates, but are listed at the bottom of the Giving Portal
+// in the "Other donation opportunities" section (after the election candidates).
+export const otherDonationOpportunities = [
+  {
+    _id: '1',
+    name: 'Centre for Effective Altruism',
+    description: 'CEA helps to build and nurture the EA community; running EA conferences, the EA Forum, supporting groups, writing newsletters and content, and more.',
+    logoSrc: 'https://res.cloudinary.com/cea/image/upload/q_auto,f_auto/v1701460414/Screen_Shot_2023-12-01_at_2.53.11_PM',
+    href: 'https://www.centreforeffectivealtruism.org/',
+    fundraiserLink: 'https://www.centreforeffectivealtruism.org/donate',
+  },
```

Comment:
I would rather add them to the db with a flag indicating they aren't election candidates, it seems fairly likely we'll want to edit the descriptions etc. But I'll merge this and change it if we actually do get a request to edit them

Reply:
I think there were a handful of required fields on the candidates table that would make that slightly annoying, and I'm guessing basically no one will see this section on the forum so no reason to put more effort into it